packages/snaps-sdk/src/jsx/validation.ts

```diff
@@ -815,6 +837,21 @@ export const BoxChildStruct = typedUnion([
   AvatarStruct,
 ]);
 
+export const NotificationComponentsStruct = typedUnion([
```

Comment:
What are we trying to exclude here?

Reply:
Any of the components not on this list haha

Reply:
I was looking for more context as to why we are doing this, what is the goal here by excluding which components?

Reply:
I think we wanted to make sure we excluded form components, the idea is to only allow some presentational components and not anything that changes state.

Reply:
We decided to open jsx notifications to any element: [213360a](https://github.com/MetaMask/snaps/pull/2881/commits/213360a5a3cc02023a47c894159c52fca017ef94)